src/windows/WindowsAccessibilityUtil.ts

```diff
@@ -0,0 +1,20 @@
+/**
+* WindowsAccessibilityUtil.ts
```

Comment:
See note above. This file and its exported class shouldn't start with "Windows".